Rmarkdown/Multiple_Ascending_Dose_PKPD_continuous.Rmd

```diff
@@ -220,61 +224,76 @@ gg <- gg + guides(color = guide_legend(""), fill = guide_legend(""))
+
+gg1 <- gg + ggtitle("Concentration on Linear Scale")
+gg2 <- gg + xgx_scale_x_log10() + ggtitle("Concentration on Log Scale")
+
+grid.arrange(gg1,gg2,nrow = 1)
 ```
 
-```{r, error = TRUE, cache = TRUE, fig.width = 8, fig.height = 3, warning = FALSE, message= FALSE}
+### PD marker by Concentration, faceted by visit, mean (95% CI) by concentration bins
+
+```{r, error = TRUE, cache = TRUE, fig.width = 8, fig.height = 6, warning = FALSE, message= FALSE}
 data_to_plot <- pkpd_data_wide %>% subset(PROFDAY %in% PD_PROFDAYS,)
 
 gg <- ggplot(data = data_to_plot,
-             aes(x = CONC, y = PD, color = TRTACT_high2low))
+             aes(x = CONC, y = PD))
 gg <- gg + xgx_annotate_status(status)
-gg <- gg + geom_point()
+gg <- gg + geom_point(aes(color = TRTACT_high2low))
 gg <- gg + geom_point(data = data_to_plot %>% subset(CENS==1,), color="red", shape = 8)
 gg <- gg + guides(color = guide_legend(""), fill = guide_legend(""))
 gg <- gg + labs(x = conc_label , y = pd_label)
-gg <- gg + xgx_scale_x_log10() 
-gg + facet_grid(~DAY_label)
+gg <- gg + xgx_stat_ci(bins = 4, geom = "point", shape = 0, size = 4)
+gg <- gg + xgx_stat_ci(bins = 4, geom = "errorbar")
+gg <- gg + facet_grid(~DAY_label)
+
+gg1 <- gg + xgx_geom_smooth_emax(color = "black") + ggtitle("Concentration on Linear Scale")
+gg2 <- gg + xgx_scale_x_log10() + ggtitle("Concentration on Log Scale")
+
+grid.arrange(gg1,gg2,nrow = 2)
+
 ```
 
 <!--START_EXPLANATION-->
 Plotting AUC vs response instead of concentration vs response may make more sense in some situations. For example, when there is a large delay between PK and PD it would be diffcult to relate the time-varying concentration with the response. If rich sampling is only done at a particular point in the study, e.g. at steady state, then the AUC calculated on the rich profile could be used as the exposure variable for a number of PD visits. If PK samples are scarce, average Cmin could also be used as the exposure metric.
 <!--END_EXPLANATION-->
-```{r, error = TRUE, cache = TRUE, fig.width = 8, fig.height = 6, warning = FALSE, message= FALSE}
-NCA_plot = NCA %>%
-  group_by(PARAM) %>%
-  mutate(VALUE_QUART = cut(VALUE, quantile(VALUE, na.rm=TRUE), na.rm=TRUE, include.lowest = TRUE)) %>%
-  group_by(PARAM, VALUE_QUART) %>%
-  mutate(VALUE_MIDPOINT = median(VALUE))
 
-gg <- ggplot(data = NCA_plot, aes(x = VALUE, y = PD))
+```{r, error = TRUE, cache = TRUE, fig.width = 8, fig.height = 6, warning = FALSE, message= FALSE}
+gg <- ggplot(data = NCA, aes(x = VALUE, y = PD))
 gg <- gg + xgx_annotate_status(status)
-gg <- gg + geom_point(aes(color = TRTACT_high2low)) + geom_smooth(color = "black")
-gg <- gg + xgx_stat_ci(aes(x = VALUE_MIDPOINT, y = PD), geom = "errorbar")
-gg <- gg + xgx_stat_ci(aes(x = VALUE_MIDPOINT, y = PD), geom = "point", shape = 0, size = 4)
+gg <- gg + geom_point(aes(color = TRTACT_high2low)) 
+gg <- gg + xgx_geom_smooth_emax(color = "black")
+gg <- gg + xgx_stat_ci(bins = 4, geom = "errorbar")
```

Comment:
@margoal1 - use of xgx_stat_ci() in a lot of these plots is a bit strange to me and my first thought is that it should be deleted.  It makes sense for logistic regressions, where the fit curve is continuous between zero and one, but each data point can only be zero or one.  But for continuous dependent variable, I'm not sure what the point is of doing the binning summary.  Can't you just look and see if the curve seems to go through the points?

Reply:
Including summaries binned by concentration quantile helps to detect model misspecification, and is a recommended plot for exposure response analyses (e.g. see https://link.springer.com/article/10.1007/s10928-017-9558-5/tables/5, last item in table). It's not always easy to look and see if the curve seems to go through the points. The existence of the smooth line can make you believe that it is going through the data well, when it actually isn't. Whether or not the curve goes through the data can be more difficult to determine with larger datasets. Additionally, we want the users of xgx to have multiple options for exploring their data, including both xgx_stat_ci and xgx_geom_smooth_emax in the one plot is an efficient way to provide users with both of these options. They may choose not to do a smooth emax fit at all, in which case they could use the xgx_stat_ci instead.

Reply:
Cool, thanks for sharing the reference!